magic/ext/oauth/build.gradle

```diff
@@ -37,8 +37,8 @@ dependencies {
     implementation 'androidx.browser:browser:1.4.0'
     implementation "com.google.code.gson:gson:$gson_version"
     implementation 'org.jetbrains.kotlin:kotlin-reflect:1.7.10'
-//    implementation project(':magic:core')
-    implementation 'link.magic:magic-android:[4.0,5.0['
+    implementation project(':magic:core')
+//    implementation 'link.magic:magic-android:[4.0,5.0['
```

Comment:
nit: do we need to keep this commented out code?

Reply:
Good catch